src/components/Dashboard/Dashboard.story.jsx

```diff
@@ -0,0 +1,161 @@
+import { storiesOf } from '@storybook/react';
+import { Button } from 'carbon-components-react';
+
+import {
+  DASHBOARD_BREAKPOINTS,
+  CARD_DIMENSIONS,
+  ROW_HEIGHT,
+  CARD_SIZES,
+  CARD_TYPES,
+} from '../../constants/LayoutConstants';
+
+import Dashboard from './Dashboard';
+
+const originalCards = [
+  {
+    title: 'SMALL',
+    id: 'facilitycard',
+    size: CARD_SIZES.SMALL,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+  {
+    title: 'XSMALL',
+    id: 'facilitycard-xs',
+    size: CARD_SIZES.XSMALL,
+    type: CARD_TYPES.VALUE,
+    content: [{ title: 'Comfort Level', value: 89, unit: '%' }],
+  },
+  {
+    title: 'XSMALL',
+    id: 'facilitycard-xs2',
+    size: CARD_SIZES.XSMALL,
+    type: CARD_TYPES.VALUE,
+    content: [{ title: 'Comfort Level', value: 89, unit: '%' }],
+  },
+  {
+    title: 'XSMALL',
+    id: 'facilitycard-xs3',
+    size: CARD_SIZES.XSMALL,
+    type: CARD_TYPES.VALUE,
+    content: [{ title: 'Comfort Level', value: 89, unit: '%' }],
+  },
+  {
+    title: 'TALL',
+    id: 'facilitycard2',
+    size: CARD_SIZES.TALL,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+  {
+    title: 'MEDIUM',
+    id: 'facilitycard3',
+    size: CARD_SIZES.MEDIUM,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+  {
+    title: 'MEDIUM',
+    id: 'facilitycard4',
+    size: CARD_SIZES.MEDIUM,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+  {
+    title: 'WIDE',
+    id: 'facilitycard5',
+    size: CARD_SIZES.WIDE,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Heat', value: 1976, unit: 'K' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+  {
+    title: 'LARGE',
+    id: 'facilitycard6',
+    size: CARD_SIZES.LARGE,
+    type: CARD_TYPES.VALUE,
+    content: [
+      { title: 'Comfort Level', value: 89, unit: '%' },
+      { title: 'Utilization', value: 76, unit: '%' },
+      { title: 'Number of Alerts', value: 17 },
+    ],
+  },
+];
+
+const StatefulDashboard = ({ ...props }) => {
+  const [cards, setCards] = useState(originalCards);
+
+  const handleAdd = () => {
+    setCards([
+      ...cards,
+      {
+        title: 'SMALL', // faker.company.companyName(),
```

Comment:
I think we should be using `faker` in all users inputs.

Reply:
I think on this one we want a specific title to match the card size.  Can we leave the comments for now in case we need to add debugging and remove once we move out of experimental?